.jshintrc

```diff
@@ -12,6 +12,7 @@
   "undef": true, // Require all non-global variables be declared before they are used.
   "unused": false, // Warn unused variables.
   "strict": true, // Require `use strict` pragma in every file.
+  "quotmark": "single", // Require only single quotes
```

Comment:
@lirantal and @ilanbiala , can we keep quotmark in here until #763 is addressed? Reason being is that I missed a lot of the double quote issues in my last PR.